cmd/unused-exporter/exporter.go

```diff
@@ -286,10 +291,12 @@ func lastUsedTS(d unused.Disk) float64 {
 }
 
 func getRegionFromZone(p unused.Provider, z string) string {
-	if strings.ToLower(p.Name()) == "azure" {
+	switch p.Name() {
+	case providerAWS :
+		return z[:len(z)-1]
+	case providerGCP:
+		return z[:len(z)-2]
+	default:
```

Comment:
I think this warrants adding some tests to ensure a regression isn't introduced and some comments with maybe an example so its' clear.

Reply:
nevermind, it's already legacy, jjo sneaked [this one](https://github.com/grafana/unused/commit/b728b1abfe7fe6bc1f3abaea02b856ead0746520) in yesterday 🤣 lemme give it another spin